### PR TITLE
Update katello rpm signing docs

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by ehelms** [comment]
Thanks for the updates. If we were going to automate this in the future would you see this as a tool_belt command or obal functionality?

**Comment by ekohl** [comment]
Something like tool_belt would make sense to me because the Foreman process is *very* similar, just with different tags/versions.

**Comment by jlsherrill** [comment]
yep +1 to toolbelt

File: plugins/katello/developers.md [hunk 1]
```diff
@@ -498,17 +501,18 @@ done;
 Get (decrypt) Katello key and gpg --import it. Of course, you need to be a person who knows how to decrypt the key. Decrypt the passphrase as well. The key that we've used starting with Katello 1.0 is D5A88496. Before doing the first sign, import katello key. In our git repo with keys do this:
 
 ```
-cd gpg-keys/katello-private
+cd gpg-keys/katello-private/$VERSION
 decrypt.sh
-gpg --import katello.private.asc
+gpg --improt katello.private.asc
```
Comment (ekohl) [inline plugins/katello/developers.md:506]: improt -> import

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by jlsherrill** [comment]
Closing, as i do have a script that i want to adapt to toolbelt

**Pull request closed without merging**